dice1.js

```diff
@@ -0,0 +1,35 @@
+const rollDice = function(){
+    const MyRandom1 = Math.floor(Math.random() * 6);
+    const MyRandom2 = Math.floor(Math.random() * 6);
+    console.log("Dice rolled are "+MyRandom1 +" and "+MyRandom2);
+    console.log("sum is "+ (MyRandom1+MyRandom2));
+}
+rollDice();
+
+//bonus
+
+const rollDice1 = function (num) {
+    const MyRandom1 = Math.floor(Math.random() * 6);
```

Comment:
this code doesn't achieve the bonus object, it only does three rolls.
thank you for the effort.